nncf/experimental/onnx/graph/model_transformer.py

```diff
@@ -11,118 +11,106 @@
  limitations under the License.
 """
 
-from typing import TypeVar
-
 from copy import deepcopy
-import onnx
+from typing import List, TypeVar
 
-from nncf.common.graph.model_transformer import ModelTransformer
+import onnx
+from nncf.common.graph.definitions import NNCFGraphNodeType
+from nncf.common.graph.graph import NNCFGraph
 from nncf.common.graph.transformations.commands import TargetType
 from nncf.common.quantization.structs import QuantizationMode
-from nncf.common.graph.definitions import NNCFGraphNodeType
 from nncf.common.utils.logger import logger as nncf_logger
-from nncf.experimental.onnx.graph.onnx_graph import ONNXGraph
 from nncf.experimental.onnx.graph.nncf_graph_builder import GraphConverter
-from nncf.experimental.onnx.graph.transformations.layout import ONNXTransformationLayout
-from nncf.experimental.onnx.graph.transformations.commands import ONNXQuantizerInsertionCommand
+from nncf.experimental.onnx.graph.onnx_graph import ONNXGraph
 from nncf.experimental.onnx.graph.transformations.commands import ONNXOutputInsertionCommand
+from nncf.experimental.onnx.graph.transformations.commands import ONNXQuantizerInsertionCommand
+from nncf.experimental.onnx.graph.transformations.layout import ONNXTransformationLayout
+from nncf.experimental.post_training.compression_builder import MODEL_TRANSFORMERS
+from nncf.experimental.post_training.graph.model_transformer import ModelTransformerMeta
 
 ModelType = TypeVar('ModelType')
```

Comment:
Do we really need a template variable here? Since this model transformer is concerned with ONNX models only, maybe it is better to just use `onnx.ModelProto` type hints.

Reply:
Removed